extensions/reactive-pg-client/deployment/src/main/java/io/quarkus/reactive/pg/client/deployment/ReactivePgClientProcessor.java

```diff
@@ -37,7 +38,8 @@ NativeImageConfigBuildItem config() {
 
     @BuildStep
     @Record(ExecutionTime.RUNTIME_INIT)
-    void build(BuildProducer<FeatureBuildItem> feature, BuildProducer<PgPoolBuildItem> pgPool, PgPoolRecorder recorder,
+    ServiceStartBuildItem build(BuildProducer<FeatureBuildItem> feature, BuildProducer<PgPoolBuildItem> pgPool,
+            PgPoolRecorder recorder,
```

Comment:
Shouldn't we do the same in `ReactiveMySQLClientProcessor` ?

Reply:
IDK. I was asked to fix this one ;-). But if it's using the same pattern then yes... I'll take a look.